apps/wing/package.json

```diff
@@ -34,6 +34,7 @@
     "@winglang/compiler": "workspace:^",
     "@winglang/sdk": "workspace:^",
     "chalk": "^4.1.2",
+    "chokidar": "^3.5.3",
```

Comment:
Is this the same watch library wing console used?

@skyrpex ?

We should be very careful to add a dependency to our cli. Let's make sure this is absolutely needed.

Reply:
It's the same one, so it doesn't broaden our dependency closure